_posts/en/newsletters/2022-11-09-newsletter.md

```diff
@@ -0,0 +1,330 @@
+    affected BTCD and LND again, but it also affected at least [two
+    other][liquid and rust bitcoin vulns] significant projects or
+    services.  All users of affected systems should upgrade immediately.
+    We repeat our advice from three weeks ago for anyone using any
+    Bitcoin software sign up for security announcements from that
+    software’s development team.
+
+    With the release of this newsletter, Optech has also added a special
+    topic page where we list the names of [the amazing people who
+    responsibly disclosed a vulnerability][topic responsible
+    disclosures] that we've summarized in an Optech newsletter.  We
+    extend our most heartfelt thanks to everyone listed.  There are
+    likely several other disclosures not listed because they haven't
+    been made public yet.  We look forward to thanking those discoverers
+    and adding them to the topic page of Bitcoin security disclosure
+    heroes.  We of course also thank all the reviewers of proposals and
+    pull requests whose diligent effort prevented innumerable
+    security bugs from making it into released software.
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review Club][]
+meeting, highlighting some of the important questions and answers.  Click on a
+question below to see a summary of the answer from the meeting.*
+
+[Relax MIN_STANDARD_TX_NONWITNESS_SIZE to 65 non-witness bytes][review club 26265]
+is a PR by instagibbs that relaxes the mempool policy's non-witness transaction size
+constraints. It allows transactions to be as small as 65 bytes, replacing
+the current policy that requires transactions to be at least 85 bytes.
+
+Since this review club meeting, this PR has been closed in favor of
+[PR 26398](https://github.com/bitcoin/bitcoin/pull/26398), which
+relaxes policy even further by disallowing _only_ 64-byte transactions.
+The relative merits of these two slightly-different policies were
+discussed during review club.
+
+{% include functions/details-list.md
+  q0="Why was the minimum transaction size 82 bytes?
+  Can you describe the attack?"
+
+  a0="The 82-byte minimum, which was introduced by PR 11423 in 2018, is
+  the size of the smallest standard payment transaction. This was
+  presented as a cleanup of the standardness rules. But in reality,
+  the change was to prevent a 64-byte transaction from being considered standard,
+  because this size allowed a
+  [spoof payment attack](https://github.com/advisories/GHSA-v55p-4chq-6grj)
+  against SPV clients (making them think they've received a payment when they hadn't).
+  The attack involves tricking an SPV client into thinking that a 64-byte
+  transaction is an inner node of the transaction merkle tree, which is also
+  64 bytes in length."
+
+  a0link="https://bitcoincore.reviews/26265#l-35"
+
+  q1="A participant asked, was it was necessary to fix this vulnerability
```

Comment:
Not a huge fan of this question (mostly the answer)

Reply:
I like the question, because it's exactly what I thought before the meeting (when I read how expensive the attack would be). But formulating the answer as another question isn't good (I think that's what you're objecting to); would this be better?
>  a1="There was some agreement, but one participant pointed out that our intuition about this could be wrong."

Or I would be fine with removing the question entirely. It is true that the answer is pretty brief, especially compared to most answers. But really nothing else was said about this question during the meeting.

Reply:
+1 for keeping this question.  I think it's a really good question given that the vulnerability was kept private for a long time before its accidental disclosure.  We definitely want to encourage developers to look back in hindsight and determine whether that was the best course of action, and be prepared to use their individual conclusions when evaluating future security issues they come across.

Reply:
Good points; this was actually the first time I formulated a question here for the newsletter that wasn't in the host's pre-meeting list of questions. But if a question is asked during the meeting that can provide good content here, seems to me we can include it here.